docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -228,6 +229,10 @@ This property is optional; default is ``https``.
 Internal columns
 ----------------
 
+Note: internal column prefix is configurable by ``kafka.internal-column-prefix``
+configuration property and defaults to ``_``. If you have configured the
+property to a different value the column names below might appear different.
```

Comment:
A different prefix affects the internal column names in the following sections. For example, a value of `internal_` changes the partition ID column name from `internal_partition_id` to `internal_partition_id`.